### PR TITLE
Better error handling

### PR DESCRIPTION
**Pull request by jmcs**

### CODE REVIEW

File: lizzy/api.py [hunk 1]
```diff
@@ -89,18 +96,19 @@ def create_stack(new_stack: dict) -> dict:
                    'parameters': parameters}
     tags = {'LizzyKeepStacks': keep_stacks,
             'LizzyTargetTraffic': new_traffic}
-    if senza.create(senza_yaml, stack_version, image_version, parameters,
-                    disable_rollback, tags):
+    try:
+        senza.create(senza_yaml, stack_version, image_version, parameters,
+                     disable_rollback, tags)
+    except ExecutionError as error:
```
Comment (rafaelcaricio) [inline lizzy/api.py]: Wouldn't that be a case to let this exception be handled by `exception_to_connexion_problem`? Seems like now we have inconsistence with throwing 400 and 500 errors for `ExecutionError` exceptions.

Comment (jmcs) [inline lizzy/api.py]: In this case if an execution error happens it's mostly likely caused by a bad request not a server error.

Comment (rafaelcaricio) [inline lizzy/api.py]: Ok. Makes sense.

**Comment by rafaelcaricio** [comment]
👍

**Pull request merged**